src/extensions/default/bramble/lib/RemoteCommandHandler.js

```diff
@@ -25,6 +25,7 @@ define(function (require, exports, module) {
     var Tutorial = require("lib/Tutorial");
     var Theme = require("lib/Theme");
     var UI = require("lib/UI");
+    var PreviewView = require("lib/iframe-browser");
```

Comment:
let's follow the convention brackets has established and use the name of the module as the variable name, i.e. `IframeBrowser`

Reply:
Did this change locally

Reply:
Can you push these changes up here too please?